packages/core/src/crypto/JwsService.ts

```diff
@@ -123,3 +159,17 @@ export interface VerifyJwsResult {
   isValid: boolean
   signerVerkeys: string[]
 }
+
+export interface Jwk {
+  kty: string
```

Comment:
Might be fine, but do we have a list of key types we support? Would be good to only allow those strings.